README.md

```diff
@@ -19,6 +19,23 @@ conda install -c conda-forge --strict-channel-priority xdem
 ```
 The `--strict-channel-priority` flag seems essential for Windows installs to function correctly, and is recommended for UNIX-based systems as well.
 
+Solving dependencies can take a long time with `conda`. To speed up this, consider installing `mamba`:
```

Comment:
Fully agreed, there is quite a bit of work on the README to make it a more agreeable read for users. I think we should focus more on the DEM thematic and a couple examples with clear links to documentation rather than all the coding stuff (how to contribute, how to install, etc... should live in separate files simply referred in the README).